examples/text_classification/run_text_classifier.py

```diff
@@ -36,6 +36,7 @@ def setup_model(device, model_setup):
 
 def run_online(device):
     # predict labels online
+    print("Enter inputs for Online Predictions")
```

Comment:
Could you use single quotes, per convention?

Reply:
Okay I'll change to single quotes and commit again